lib/tasks/load_blacklist.rake

```diff
@@ -1,4 +1,5 @@
-# Black list is generated by running aws_batch/black_list.py |cut -f2 -d ',' |sort|uniq > blacklist.txt
+# Black list is generated by running
+#  grep -i ',artificial sequences,' lineages.csv |cut -f 2 -d','  >> config/taxon_blacklist.txt
```

Comment:
should we add this command to the indexing generation scripts and download the file from the taxonomy S3 location rather than checking it in?

Reply:
sure. this is a pretty small file though so I think it's fine to keep it in the code base. but i will make a note.